views/components/collapsibleBlock.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * All About Olaf
+ * Individual row for inside a collapsible view
+ */
+
+import React from 'react'
+import {
+  View,
+  TouchableOpacity,
+} from 'react-native'
+
+import Collapsible from 'react-native-collapsible'
+
+
+export default class CollapsibleBlock extends React.Component {
+
+  static propTypes = {
+    borderColor: React.PropTypes.string.isRequired,
+    children: React.PropTypes.array.isRequired,
```

Comment:
`children` is better typed as `React.PropTypes.node`, as `this.props.children` is not in an array [if there's only one child](https://facebook.github.io/react/tips/children-props-type.html). `node` covers anything React can render, so elements, text, and arrays of the above.